context-monitoring/src/main/java/de/atb/context/monitoring/monitors/file/FilePairSystemMonitor.java

```diff
@@ -260,17 +254,12 @@ protected final void handleWatchEvents(final List<WatchEvent<?>> events,
                           + context.toString();
             InterpreterConfiguration setting = this.interpreter
                 .getConfiguration(file);
-            if (e.kind() == StandardWatchEventKind.ENTRY_CREATE) {
+            if (e.kind() == StandardWatchEventKinds.ENTRY_CREATE) {
                 fileCreated(file, time, setting);
-            } else if (e.kind() == StandardWatchEventKind.ENTRY_MODIFY) {
+            } else if (e.kind() == StandardWatchEventKinds.ENTRY_MODIFY) {
                 fileModified(file, time, setting);
-            } else if (e.kind() == StandardWatchEventKind.ENTRY_DELETE) {
+            } else if (e.kind() == StandardWatchEventKinds.ENTRY_DELETE) {
                 fileDeleted(file, time, setting);
-            } else if (e.kind() == ExtendedWatchEventKind.ENTRY_RENAME_FROM) {
```

Comment:
I assume that `ENTRY_RENAME_FROM` and `ENTRY_RENAME_TO` are now covered by `ENTRY_CREATE` and `ENTRY_DELETE`? I.e. renaming a file would trigger an `ENTRY_DELETE` and then an `ENTRY_CREATE` event?

So file renaming doesn't have t be handled separately, like before?

Comment also applies to the other two file monitor classes.

Reply:
Does not need to be handled separately. 
Downside, a renamed file will be analysed again (because of the new name).
The old delete file (-name) will be removed from the analysed files list